velociraptor/tools/histogram.py

```diff
@@ -60,6 +69,18 @@ def create_histogram_given_bins(
     histogram.name = "Number of Haloes"
     bin_centers.name = masses.name
 
+    # Compute cumulative sum?
+    if cumulative:
+        if reverse:
+            # Cumulative sum from high to low
+            histogram = np.cumsum(histogram[::-1])[::-1]
```

Comment:
I received a suggestion to not use the binning for cumulative histograms. That will allow us to better see where the objects are. What do you think @JBorrow ?

To implement this, all we need is an array of masses (or different quantity), and applied to it
`np.arange(start=1)`

Reply:
The problem with that is that it will make the .yml files way too big and unweidly if there are a lot of haloes.